clrtrust

```diff
@@ -96,102 +99,88 @@ EOF
 }
 
 cmd_check() {
-    local opt_verbose
-    local ret # return value
-
-    ret=0
 
     while [ $# -gt 0 ]; do
         case $1 in
-            -h|--help)
+            ("-h"|"--help")
                 print_check_help
                 return 0
                 ;;
-            -v|--verbose)
-                opt_verbose=1
+            ("-v"|"--verbose")
+                VERBOSE=1
                 shift
                 ;;
-            *)
+            (*)
                 print_check_help
                 return $EINVAL
                 ;;
         esac
     done
 
-    if [ ! -z "${VERBOSE}" ]; then
-        opt_verbose=1
-    fi
-
     if [ -e "${CLR_LOCAL_TRUST_SRC}" ]; then
         if [ ! -d "${CLR_LOCAL_TRUST_SRC}" ]; then
             1>&2 echo "${CLR_LOCAL_TRUST_SRC} must be a directory."
-            test ! -z $opt_verbose && 1>&2 cat <<EOF
-    ${CLR_LOCAL_TRUST_SRC} is used by ${BASENAME} to store local (user-added) trust and
-    distrust data.
-EOF
-            ret=$EBADST
+            verbose_error       \
+"%4s${CLR_LOCAL_TRUST_SRC} is used by ${BASENAME} to store local (user-added)
```

Comment:
Not a great fan of these uses of %4s to print out nothing in a field with width 4.

As a justification for changing it, one shouldn't pass a user constructed format string - consider what happens if CLR_LOCAL_TRUST_SRC or BASENAME has a % sign in it. 

So I would prefer to see

```
                                   verbose_error "    %s\n" \
"${CLR_LOCAL_TRUST_SRC} is used by ${BASENAME} to store local (user-added)" \
"trust and distrust data"